.scalafix3.conf

```diff
@@ -1,5 +1,6 @@
 rules = [
-  OrganizeImports
+  OrganizeImports,
+  ExplicitResultTypes
```

Comment:
as soon as the build is upgraded to 3.3.4, RemoveUnused can be used and that custom `.scalafix3.conf` can be removed